src/Microsoft.ML.TorchSharp/NasBert/BaseModule.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using TorchSharp;
+
+namespace Microsoft.ML.TorchSharp.NasBert
+{
+    internal abstract class BaseModule : torch.nn.Module
```

Comment:
>torch.nn.Module [](http://example.com/codeflow?start=41&length=15)

nit: would be good having some comment about the name casing in general telling about the torchsharp/pytorch casing?

Reply:
They have a section talking about that in the TorchSharp repo. I am happy to add a link to that. Where in our code do you think I should add it (cause I don't think it makes sense to add it to every class)?